### ISSUE TITLE
AssertionError: must be called on client thread

### ISSUE THREAD

**Issue by emerald000** [dillydill123/inventory-setups#45]
How to reproduce:

1- Run Runelite with assertions enabled.
2- Open you bank.
3- Click on the eye icon to view a setup.

You will receive that exception and the bank will not filter the items. Pressing the keybind to filter the bank works correctly. Full stacktrace:

```
ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: must be called on client thread
	at client.jt(client.java:3776)
	at client.setVarbitValue(client.java:14464)
	at client.setVarbit(client.java:50767)
	at inventorysetups.InventorySetupsPlugin.doBankSearch(InventorySetupsPlugin.java:400)
	at inventorysetups.ui.InventorySetupPluginPanel.setCurrentInventorySetup(InventorySetupPluginPanel.java:519)
	at inventorysetups.ui.InventorySetupStandardPanel$10.mousePressed(InventorySetupStandardPanel.java:500)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.Component.processMouseEvent(Component.java:6536)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6304)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4532)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:760)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:733)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:730)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by dillydill123** [comment]
Hey thanks for finding this, I think I’ve taken care of it.

**Comment by dillydill123** [comment]
Fixed in 1.7 (i hope)
